CONTRIBUTING.md

```diff
@@ -94,8 +94,22 @@ When a commit is made on the `master` branch, documentation is automatically bui
 
 ## JavaScript dependencies
 
-For the time being, we are committing JavaScript dependencies to the repository as that makes installation much easier.
-If you need to add a new library, or update the version of a library, you will need to have `npm` installed (on OS X, you can install it with Homebrew, and on Linux, you can install it with the relevant package manager).
+For the time being, we are committing JavaScript dependencies to the repository as that makes installation much easier. However there is an unbundled dependency on PhantomJS that must be installed separately. If you have `npm` installed, you can install PhantomJS by running:
```

Comment:
Hmm, sorry I didn't realize this earlier, but I think this should probably actually go earlier in this document. PhantomJS is javascript but it's a requirement for testing, rather than for the formgrader, so it should go under the "Development installation" section. I also don't know if we want to require people to install npm -- we can list it as one way to install it, but I think we should also include `apt-get install phantomjs` and `brew install phantomjs` as options.